### ISSUE TITLE
Feature Request: Allow Sessions to be moved to a Bookmark Folder

### ISSUE THREAD

**Issue by sahil48** [tim-we/tabs-aside#56]
While it is possible to do this in the bookmark manager, I feel it would be useful to allow tab sessions set aside to be moved or merged inside a bookmark folder. You can keep setting bookmarks aside to your tab aside session, and then permanently store them in a bookmark folder outside the tabs aside folder.

**Comment by tim-we** [comment]
Not sure I understand what you are asking for. Merging sessions is IMO an important feature that should be supported in the future. Tabs Aside 3 will make this a bit easier as you can just drag and drop multiple tabs between open sessions. However this should be supported in the sidebar as well.

**Comment by ootada** [comment]
I was thinking about an option to "archive a session", which would move session tabs/bookmarks to lets say a different place in bookmark tree and would not show in the list anymore.

then its up to you to manage these bookmarks.

it should also be possible to "unarchive a session" (manually moving the bookmarks should work?)

**Comment by tim-we** [comment]
>manually moving the bookmarks should work?

Yes that works although the sidebar will not update (has to be manually closed & reopened).
